paracelsus/transformers/mermaid.py

```diff
@@ -29,15 +30,16 @@ def _column(self, column: Column) -> str:
                 column_str += " PK"
         elif len(column.foreign_keys) > 0:
             column_str += " FK"
+            options.append(f"Foreign key references {column.table.name}")
+        elif column.unique:
+            column_str += " UK"
 
-        options = []
+        if column.comment:
+            options.append(f"{column.comment}")
```

Comment:
Since column.comment is a string the f string and quotes are redundant-

In other words both of these lines are the same, so it should use the succinct one:

```python
options.append(column.comment)
options.append(f"{column.comment}")
```